code/modules/power/apc.dm

```diff
@@ -146,6 +146,14 @@
 	req_access = list(ACCESS_SYNDICATE)
 	report_power_alarm = FALSE
 
+/obj/machinery/power/apc/syndicate/off/Initialize(mapload)
+	environ = 0
+	equipment = 0
+	lighting = 0
+	operating = FALSE
+	..()
+	cell.charge = 0
```

Comment:
```suggestion
/obj/machinery/power/apc/syndicate/off/Initialize(mapload)
	. = ..()
	environ = FALSE
	equipment = FALSE
	lighting = FALSE
	operating = FALSE
	cell.charge = 0
```

Parent call must be first in Initialize()

Reply:
Whoops, forgot to mention, fixed in 8a5bd102f9218a8e58928854b38e28cb980370f2